### PR TITLE
Consolidate usage of tier names in the repo

### PR DESCRIPTION
**Pull request by kuszz**
Fixes https://github.com/Alluxio/new-contributor-tasks/issues/586

### CODE REVIEW

File: core/common/src/main/java/alluxio/Constants.java [hunk 1]
```diff
@@ -206,5 +206,10 @@
   public static final String TABLE_MASTER_CLIENT_SERVICE_NAME = "TableMasterClient";
   public static final long TABLE_MASTER_CLIENT_SERVICE_VERSION = 1;
 
+  // Medium name
+  public static final String MEDIUM_MEM = "MEM";
+  public static final String MEDIUM_SSD = "SSD";
+  public static final String MEDIUM_HDD = "HDD";
```
Comment (apc999) [inline core/common/src/main/java/alluxio/Constants.java:212]: @LuQQiu @ggezer  shall we create a static class `Medium.java` and make three static member String constants called `MEM`, `SSD`, `HDD` ?  I slightly incline for this approach as we can make more fancy changes in the future in this class. But I am also cool for the current solution.. I will leave this up to you

**Review by maobaolong — CHANGES_REQUESTED** [review]
@kuszz Thank you for your contribution
 `/usr/src/alluxio/core/common/src/test/java/alluxio/wire/BlockMasterInfoTest.java:15: error: 'com.fasterxml.jackson.databind.ObjectMapper' should be separated from previous import group.`

From the build log, you can find that there are a style issue, please take a look.

**Comment by kuszz** [comment]
> @kuszz Thank you for your contribution
> `/usr/src/alluxio/core/common/src/test/java/alluxio/wire/BlockMasterInfoTest.java:15: error: 'com.fasterxml.jackson.databind.ObjectMapper' should be separated from previous import group.`
> 
> From the build log, you can find that there are a style issue, please take a look.

hi, i have modified the code for the style issue, please take a look.

**Comment by maobaolong** [comment]
@kuszz There is a conflicting file issue, please take a look, thank you.

**Comment by LuQQiu** [comment]
@kuszz the following commands help solve the conflicts:
```
git fetch upstream
git merge upstream/aster
git commit -am "before solving conflicts"
# make actual changes to solve the conflicting files
git commit -am "solve conflicts"
git push origin iss586
```

**Comment by LuQQiu** [comment]
```
Error: 3.022 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check (checkstyle) on project alluxio-core-server-master: Failed during checkstyle execution: There are 54 errors reported by Checkstyle 6.11.2 with /usr/src/alluxio/build/checkstyle/alluxio_checks.xml ruleset. -> [Help 1]
Error: 3.023 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check (checkstyle) on project alluxio-job-server: Failed during checkstyle execution: There is 1 error reported by Checkstyle 6.11.2 with /usr/src/alluxio/build/checkstyle/alluxio_checks.xml ruleset. -> [Help 1]
Error: 3.024 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check (checkstyle) on project alluxio-core-server-worker: Failed during checkstyle execution: There are 37 errors reported by Checkstyle 6.11.2 with /usr/src/alluxio/build/checkstyle/alluxio_checks.xml ruleset. -> [Help 1]
```
@kuszz Please make sure `mvn checkstyle:checkstyle` passed

**Pull request closed without merging**